Module 12 - Governer Limits/triggers/accountTriggerGLimits.trigger

```diff
@@ -0,0 +1,44 @@
+trigger accountTriggerGLimits on Account (before delete, before insert, before update) 
+{
+    
+    List<Opportunity> opptysClosedLost = [select id, name, closedate, stagename from Opportunity
+    where accountId IN :Trigger.newMap.keySet() and StageName='Closed - Lost'];
+    List<Opportunity> opptysClosedWon = [select id, name, closedate, stagename from Opportunity where
+    accountId IN :Trigger.newMap.keySet() and StageName='Closed - Won'];
+    // No need to create two seperate list can make one list using OR operator in Query
+    
+    for(Account a : Trigger.new)
+    {
+    for(Opportunity o: opptysClosedLost)
+    {
+    if(o.accountid == a.id)
```

Comment:
I think we missed this condition where it check if the opportunity is related to specific Account